### PR TITLE
[Workers] fix: update supported vitest version

### PR DESCRIPTION
**Pull request by Cherry**
### Summary

This simply updates the supported vitest version to 2.1.8. The Workers vitest integrations works with `2.0.x - 2.1.x`.

Paired with https://github.com/cloudflare/workers-sdk/pull/7384

### CODE REVIEW

File: src/content/docs/workers/testing/vitest-integration/get-started/write-your-first-test.mdx [hunk 1]
```diff
@@ -24,15 +24,15 @@ This guide will instruct you through installing and setting up the `@cloudflare/
...
+npm install vitest@2.1.6 --save-dev --save-exact
 npm install @cloudflare/vitest-pool-workers --save-dev
 ```
 
 The above commands will add the packages to your `package.json` file and install them as dev dependencies.
 
 :::note
 
-Currently, the `@cloudflare/vitest-pool-workers` package _only_ works with Vitest 2.0.5.
+Currently, the `@cloudflare/vitest-pool-workers` package _only_ works with Vitest 2.0.x - 2.1.x.
```
[the remark this answers is not in the document]
Comment (Cherry) [reply src/content/docs/workers/testing/vitest-integration/get-started/write-your-first-test.mdx]: not applicable to npm package names

[the remark this answers is not in the document]
Comment (Cherry) [reply src/content/docs/workers/testing/vitest-integration/get-started/write-your-first-test.mdx]: Not needed with package names

**Comment by Cherry** [comment]
This is good to merge now, with https://github.com/cloudflare/workers-sdk/pull/7384 landing in https://github.com/cloudflare/workers-sdk/releases/tag/create-cloudflare%402.33.3

File: src/content/docs/workers/testing/vitest-integration/get-started/write-your-first-test.mdx [hunk 2]
```diff
@@ -24,15 +24,15 @@ This guide will instruct you through installing and setting up the `@cloudflare/
...
+npm install vitest@2.1.8 --save-dev --save-exact
 npm install @cloudflare/vitest-pool-workers --save-dev
 ```
 
 The above commands will add the packages to your `package.json` file and install them as dev dependencies.
 
 :::note
 
-Currently, the `@cloudflare/vitest-pool-workers` package _only_ works with Vitest 2.0.5.
+Currently, the `@cloudflare/vitest-pool-workers` package _only_ works with Vitest 2.0.x - 2.1.x.
```
[the remark this answers is not in the document]
Comment (Cherry) [reply src/content/docs/workers/testing/vitest-integration/get-started/write-your-first-test.mdx]: Not needed with package names

[the remark this answers is not in the document]
Comment (bllchmbrs) [reply src/content/docs/workers/testing/vitest-integration/get-started/write-your-first-test.mdx]: Thanks for the feedback. We're updating here to better handle this kind of case.

Comment (GregBrimble) [inline src/content/docs/workers/testing/vitest-integration/get-started/write-your-first-test.mdx]: Per [@penalosa's comment on a related PR](https://github.com/cloudflare/cloudflare-docs/pull/18227#issuecomment-2529085907).

```suggestion
Currently, the `@cloudflare/vitest-pool-workers` package _only_ works with Vitest 2.0.x - 2.1.8.
```

**Review by GregBrimble — APPROVED** [review]
Thanks!

**Pull request merged**